test/bigtest/tests/coverage-tests.js

```diff
@@ -53,27 +80,58 @@ const serial = {
     startIssue: '1',
     endVolume: '79',
     endIssue: '2'
-  }]
+  }],
+  embargo: {
+    movingWallStart: { length: 730, unit: 'days' },
+    movingWallEnd: { length: 2, unit: 'years' },
+  },
+};
+
+const noMovingWallEndSerial = {
+  pti: {
+    titleInstance: {
+      id: '2613',
+      name: 'This is a serial without moving wall end',
+      type: { value: 'serial', label: 'Serial' },
+      subType: {
+        value: 'electronic',
+        label: 'Electronic'
+      }
+    }
+  },
+  coverage: [{
+    startDate: '12/06/2001',
+    endDate: '10/05/2007',
+    startVolume: '76',
+    startIssue: '1',
+    endVolume: '79',
+    endIssue: '2'
+  }],
+  embargo: {
+    movingWallStart: { length: 5, unit: 'months' },
+  },
 };
 
 // We also want to test the other two possible shapes for passing this data through to Coverage
 
 const monographEResource = {
-  _object: monograph
+  _object: monograph,
 };
 
 const serialEResource = {
-  _object: serial,
-  coverage: serial.coverage
+  _object: noMovingWallEndSerial,
+  coverage: noMovingWallEndSerial.coverage,
+  embargo: noMovingWallEndSerial.embargo,
 };
 
 const monographLine = {
-  resource: monographEResource
+  resource: monographEResource,
 };
 
 const serialLine = {
-  resource: serialEResource._object,
-  coverage: serial.coverage
+  resource: serial,
+  coverage: serial.coverage,
+  embargo: serial.embargo
```

Comment:
Tests are failing becuase the code expects embargo to be on`line.resource._object.embargo` vs `line.resource.embargo` as you have it here. The shape of the data needs to be matched with that of the backend